app-misc/lirc/lirc-0.9.4a-r2.ebuild

```diff
@@ -51,9 +51,15 @@ RDEPEND="
 	usb? ( virtual/libusb:0 )
 "
 
+PATCHES=(
+	"${FILESDIR}"/lirc-0.9.4-doc-path-fix.patch # https://bugs.gentoo.org/show_bug.cgi?id=589246 https://sourceforge.net/p/lirc/tickets/214/
+)
+
 src_configure() {
 	filter-flags -Wl,-O1
 	econf \
+		--docdir=/usr/share/doc/${PF} \
```

Comment:
I didn't know that - thanks for pointing that out. I checked and confirmed that `--docdir` doesn't need to be specified here.